src/app/courses/add-course/add-course.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-add-course',
+  templateUrl: './add-course.component.html',
+  styleUrls: ['./add-course.component.scss']
+})
+export class AddCourseComponent implements OnInit {
+
+  title: string;
+  description: string;
+  duration: string;
+  date: string;
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
```

Comment:
please, remove useless constructor and hook

Reply:
done